src/main/java/frc/robot/subsystems/PoseEstimatorSubsystem.java

```diff
@@ -61,11 +74,15 @@ public void periodic() {
     }
 
     public void updateVisionMeasurementTimestamp() {
-        timeStamp = Timer.getFPGATimestamp() - (Robot.LIMELIGHT_SUBSYSTEM.getTl() / 1000) - (Robot.LIMELIGHT_SUBSYSTEM.getCl() / 1000);
+        _timeStamp = Timer.getFPGATimestamp() - (Robot.LIMELIGHT_SUBSYSTEM_ONE.getTl() / 1000) - (Robot.LIMELIGHT_SUBSYSTEM_ONE.getCl() / 1000);
+    }
+
+    public void updateVisionMeasurmentTimestampSecondLimelight() {
+        _timeStamp2 = Timer.getFPGATimestamp() - (Robot.LIMELIGHT_SUBSYSTEM_TWO.getTl() / 1000) - (Robot.LIMELIGHT_SUBSYSTEM_ONE.getCl() / 1000);
     }
     
     public void resetOdometryPoseToLimelight() {
```

Comment:
add a parameter with type LimelightSubsystem, since that can be variable now that there are two.